### PR TITLE
[DOCS] Add condition example on using constants

### PR DESCRIPTION
**Pull request by linawolf**
Releases: main, 12.4

### CODE REVIEW

File: Documentation/Conditions/Index.rst [hunk 1]
```diff
@@ -1324,3 +1324,26 @@ usergroup()
...
+Check if a constant is set to a certain value
+---------------------------------------------
+
+TypoScript constants can be used in conditions with the 
+:ref:`Syntax <t3coreapi:typoscript-syntax-conditions-syntax>` for conditions:
+
+..  code-block:: typoscript
+    :caption: EXT:my_extension/Configuration/TypoScript/setup.typoscript
+
+    [{$tx_my_extension.settings.feature1Enabled} == 1]
```
Comment (froemken) [inline Documentation/Conditions/Index.rst:1345]: I love examples. Thank you. What about such information:

Works:
```
# constant
foo = 1

# setup
[{$foo} == 1]
```

Works:
```
# constant
foo = 1

# setup
[{$foo} == '1']
```

Works:
```
# constant
foo = 1

# setup
[{$foo} === 1]
```

Does not work:
```
# constant
foo = 1

# setup
[{$foo} === '1']
```

Works:
```
# constant
foo = bar

# setup
[{$foo} === 'bar']
```

Does not work:
```
# constant
foo = bar

# setup
[{$foo} === bar]
```

Does not work:
```
# constant
foo = bar

# setup
[{$foo} == bar]
```

Works:
```
# constant
foo = bar

# setup
[{$foo} == 'bar']
```

Works:
```
# constant
foo = bar

# setup
[{$foo} == "bar"]
```

**Review by froemken — COMMENTED** [review]
See my examples in source code

**Comment by linawolf** [comment]
Would you like to make a follow up with more examples? Once we have the example section in we can use more examples

**Review by froemken — APPROVED** [review]
I will do follow-up

**Pull request merged**